### PR TITLE
Added job title #1

### PR DESCRIPTION
**Pull request by usdevkota**
Added company name to

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,5 +7,6 @@
 </head>
 <body>
     <h1>Sana Devkota</h1>
+    <h2>SWE</h2>
```
Comment (udevkota) [inline index.html:10]: Please spell out software engineer

Comment (usdevkota) [reply index.html:10]: Can do!

**Review by udevkota — CHANGES_REQUESTED** [review]
Please do the request we agreed upon

**Review by udevkota — APPROVED** [review]
Looks great! Thanks for all of your hard work

**Pull request merged**